src/back/controllers/api/routes/categories/categories.js

```diff
@@ -9,10 +10,30 @@ const getAll = (req, res) => {
 };
 
 const save = (req, res) => {
-    // TODO: Implement save categories functionality
-    res.sendStatus(200);
+    const userId = req.user._id;
+    const categoryIds = getCategoryIds(req.body);
+
+    user.find({ _id: userId }).update({
```

Comment:
guys, same issue here : direct usage of persistence layer in controller
for example : this is the same as you use use http inside controller (angular)